src/views/FAQPage/FAQItem.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <Fragment>
+    <div class="faq-question" @click="() => toggleAccordion(question)">{{ question }}</div>
```

Comment:
the arrow function isnt required as the method will be triggered automatically via Vue.

Reply:
Oh, that's cool. Didn't know what. I'll update this.